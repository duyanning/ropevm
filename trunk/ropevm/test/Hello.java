
class Hello{
	public static void main(String[] argv){
		Util.printIntAndDouble(123, 124.125);
		Util.printString("Hello\u0080\u0800\n");
	}
}

