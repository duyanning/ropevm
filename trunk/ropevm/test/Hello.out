Int: 123; Double: 124.125000
Helloࠀ
